gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationRequestMapperTest.java

```diff
@@ -383,7 +384,7 @@ static void beforeAll() {
 
         @BeforeEach
         void beforeEach() {
-            setupMultipleOrdersToOnePlanStubs(REPEAT_PRESCRIPTION_EXTENSION);
+            setupMultipleOrdersToOnePlanStubs(REPEAT_PRESCRIPTION_EXTENSION, MedicationStatementStatus.ACTIVE, true);
```

Comment:
Repeating something I said in Teams, I find the mix between mocking at JSON + XML levels a bit jarring. I think I disagree with the testing boundaries being used 😢 .

Reply:
I hate it too, I mentioned it when I first started this ticket.  Maybe when i move it to dealing with xml I help get around this, as the testing for mapping will be done at the MedicationStatementMapper / MedicationRequestPlan tests and so a lot of these tests wont we required at the MedicationRequestMapper stage.